src/pages/bubbles.svelte

```diff
@@ -0,0 +1,7 @@
+<svelte:head>
```

Comment:
Oh, there's a page for that. Move bubbles logic here.

Reply:
moved logic into bubble file